src/useClickedOutside/useClickedOutside.ts

```diff
@@ -0,0 +1,41 @@
+import type { RefElementType } from '@muban/muban';
+import { useEventListener } from '../useEventListener/useEventListener';
+import type { DomElementOrRef, DomElementsOrRefCollection } from '../utils/util.types';
+import { getElements } from '../utils/getElements';
+
+/**
+ * A small wrapper method to easily check if a user clicked inside a HTMLElement.
+ *
+ * @param target - The target there the user clicked.
+ * @param container - The container where to check against.
+ */
+export function isClickedInside(target: EventTarget, container: RefElementType): boolean {
```

Comment:
the language is a bit confusing, the jsdoc comment mentions
> check if a user clicked inside a HTMLElement

but `target` is of type `EventTarget` ? I think it would be fine if both parameters were simply `Element`

Reply:
Updated!